### DISCUSSION TITLE
Big diference in unused packages 0.7.12 - dev-main (0.8)

### DISCUSSION THREAD

**Discussion by filipagh** [composer-unused/composer-unused#308] [Q&A]
in version  0.7.12 composer-unused found 5 packages as unused 
 * ext-intl  (i did not found any usage for it but im not sure for now)
 * ext-pdo_mysql  (many usages)
 * ezyang/htmlpurifier (many usages)
 * php-http/guzzle6-adapter (not used)
 * symfony/filesystem (not used)

but now `dev-main` report 34
<details>
  <summary>Spoiler warning</summary>
  
  ```
Unused packages
 ✗ ext-gd
 ✗ ext-exif
 ✗ ext-iconv
 ✗ ext-xml
 ✗ ext-intl
 ✗ ext-pdo_mysql
 ✗ elasticsearch/elasticsearch
 ✗ zendesk/zendesk_api_client_php
 ✗ qualityunit/resqu-client
 ✗ bandwidth-throttle/token-bucket
 ✗ archon810/smartdomdocument
 ✗ geoip2/geoip2
 ✗ jolicode/slack-php-api
 ✗ helpscout/api
 ✗ ezyang/htmlpurifier
 ✗ facebook/graph-sdk
 ✗ slim/slim
 ✗ braintree/braintree_php
 ✗ jhut89/mailchimp3php
 ✗ bjeavons/zxcvbn-php
 ✗ knplabs/github-api
 ✗ icawebdesign/hibp-php
 ✗ prestashop/prestashop-webservice-lib
 ✗ middlewares/trailing-slash
 ✗ ralouphie/mimey
 ✗ php-http/guzzle6-adapter
 ✗ getresponse/sdk-php
 ✗ pipedrive/pipedrive
 ✗ symfony/filesystem
 ✗ themattharris/tmhoauth
 ✗ enshrined/svg-sanitize
 ✗ amphp/http-client
 ✗ phpmailer/phpmailer
 ✗ chillerlan/php-qrcode

  ```
 
</details>

even i can find many usages across code, (*.class.php) 
i think `dev-main` ignore custom file autoload in composer.json or something like that

should i report it as bug or is there way to find out why they are unused ? 

Thanks

**Comment by icanhazstring** [comment]
The difference between 0.7 and 0.8 is the way composer-unused is finding files to parse.

This comes down to the projects composer.json and the autoload directive.

If your source files are coming from a different folder than that, you can add them as additional files (see readme) 

Edit:
If this doesn't help. Please file an issue with a possible replication demo project.
So I can get a better look at it.
